rpg.py

```diff
@@ -46,6 +47,12 @@ def get_all_enemies(self, players: List) -> List[int]:
             if not self.is_ally(player) and player.is_alive()
         ]
 
+    def get_all_allies(self, players: List) -> List[int]:
+        return[
+            index for index, player in enumerate(players)
+            if self.team == 1 and self.is_alive()
```

Comment:
Don't hardcode the team value.

try `if self.team == player.team` instead.

Also you are check if `self` is alive, not the other players (`player`)

Reply:
If I don't put self, it makes the method static